core/src/layers/complete.rs

```diff
@@ -806,7 +806,7 @@ mod tests {
     #[tokio::test]
     async fn test_writer() {
         let op = new_test_operator(Capability::default());
-        let res = op.write("path", vec![]).await;
+        let res = op.write("path", Vec::<u8>::new()).await;
```

Comment:
Does this change required? I expect `vec![]` can work as is.

Reply:
This should be necessary because we have implemented conversion from Vec<u8> and Vec<Bytes> for Buffer. If we directly vec![], the compiler cannot determine which one it is and will report an error.

Reply:
Nice point!

Reply:
How about changing the code here into following for better reading?

```rust
let bs: Vec<u8> = vec![];
let res = op.write("path", bs).await;
```

Reply:
Updated.